src/base/utils/misc.cpp

```diff
@@ -65,6 +65,8 @@
 #if (defined(Q_OS_UNIX) && !defined(Q_OS_MAC)) && defined(QT_DBUS_LIB)
 #include <QDBusInterface>
 #include <QDBusMessage>
+#endif
+#if defined(Q_OS_UNIX) && !defined(Q_OS_MAC)
 #include <QRegExp>
```

Comment:
QRegExp is used in Utils::Misc::parseHtmlLinks too, which is also not limited to linux only.

Reply:
Yet AppVeyor finished without error...
Anyway, I'll make necessary changes.

Reply:
It is probably because we also include `QRegularExpression`. 
I should probably drop the `QRegExpr` include, right?

Reply:
Well, I am not sure about `QRegularExpression` including `QRegExp`, but in any case we shouldn't depend on indirect includes.